components/filter/filter-overflow-group.js

```diff
@@ -27,16 +40,46 @@ class FilterOverflowGroup extends OverflowGroupMixin(RtlMixin(LitElement)) {
 		`];
 	}
 
+	constructor() {
+		super();
+
+		this._filterIds = '';
+	}
+
 	firstUpdated(changedProperties) {
 		super.firstUpdated(changedProperties);
 
 		this.addEventListener('d2l-filter-change', this._handleFilterChange);
+
+		if (!this.tags) return;
+
+		this._filterTags = document.createElement('d2l-filter-tags');
+		this.insertAdjacentElement('afterend', this._filterTags);
+	}
+
+	updated(changedProperties) {
+		super.updated(changedProperties);
+
+		if (changedProperties.has('_filterIds') && this.tags) {
+			this._filterTags.setAttribute('filter-ids', this._filterIds.trim());
+		}
 	}
 
 	convertToOverflowItem(node) {
 		const tagName = node.tagName.toLowerCase();
-		if (tagName !== 'd2l-filter') console.warn(`d2l-filter-overflow-group: ${tagName} is invalid in this group. This group should only contain d2l-filter direct child elements.`);
-		else return createFilterItem(node);
+		if (tagName !== 'd2l-filter') {
+			console.warn(`d2l-filter-overflow-group: ${tagName} is invalid in this group. This group should only contain d2l-filter direct child elements.`);
+			return;
+		}
+
+		if (this.tags && !node.hasAttribute('data-filter-tags-subscribed')) {
+			const filterId = node.id || getUniqueId();
```

Comment:
Maybe we should add an `id` to one of the `d2l-filter`s in the visual-diff tests and demo, just to help catch any possible regressions around that